packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -277,14 +279,14 @@ export class ContextualMenu extends BaseComponent<IContextualMenuProps, IContext
           hidden={ this.props.hidden }
         >
           <div
-            role='menu'
+            role={ shouldFocusOnContainer ? 'menu' : undefined }
```

Comment:
Why should the role change based on whether or not it is focusable? What gets labeled as a `menu` in the other case?

Reply:
So when focus is NOT on the container (and is instead on the first menu item), adding role='menu' to the container stopped Narrator from reading the aria-label on the menu. Currently nothing gets labeled as a menu in the case where focus is on the first menu item (and narration works optimally).